app/jobs/check_catalogues_job.rb

```diff
@@ -1,37 +1,34 @@
+require 'capybara'
+require 'capybara/poltergeist'
+require_relative '../../app/jobs/scrape_catalogues_job.rb'
+require_relative '../../app/jobs/delete_catalogues_job.rb'
+require_relative '../../app/jobs/capybara_quit_job.rb'

```

Comment:
Any luck with `require 'jobs/scrape_catalogues_job'` etc. instead of the `require_relative ...` statements?